pkg/frontend/querymiddleware/split_and_cache.go

```diff
@@ -35,7 +35,10 @@ import (
 
 const (
 	// Cache entries for 7 days. We're not disabling TTL because the backend client currently doesn't support it.
-	resultsCacheTTL = 7 * 24 * time.Hour
+	resultsCacheTTL                       = 7 * 24 * time.Hour
+	notCachableReasonUnalignedRequest     = "unaligned-request"
```

Comment:
I would clarify what is unaligned.
```suggestion
	notCachableReasonUnalignedTimeRange     = "unaligned-time-range"
```